eox_nelp/api_clients/__init__.py

```diff
@@ -20,6 +20,8 @@
 class AbstractApiClient(ABC):
     """Abstract api client class, this implement a basic authentication method and defines methods POST and GET"""
 
+    extra_headers_key = None
```

Comment:
```suggestion
    settings_extra_headers_key = None
```
?

Reply:
I understand this call.
https://github.com/eduNEXT/eox-nelp/pull/79/files/e39e58048a4bee2ce7497153ad9b93e9eaf5ade3#diff-ee7e8d8eb8c797d0a4c89e44eead86359e579a14320f30b25034c2d6807a05c2R43